tilelog/__init__.py

```diff
@@ -52,26 +52,27 @@ def cli(date, staging, generate_success, region, tile, host, app, country):
 
 def tile_logs(curs, date, dest):
     click.echo("Querying for tile usage")
-    query = """
+    query = f"""
 SELECT z, x, y,
         COUNT(*)
-FROM {tablename}
+FROM {tilelog.constants.FASTLY_PARQET_LOGS}
 WHERE year = %(year)d
     AND month = %(month)d
     AND day = %(day)d
 GROUP BY z, x, y
 HAVING COUNT(DISTINCT ip) >= %(min_distinct)d
     AND COUNT(*) >= %(min_requests)d
 ORDER BY z, x, y
-    """.format(tablename=tilelog.constants.FASTLY_PARQET_LOGS)
+    """
     curs.execute(query, {"year": date.year, "month": date.month,
                          "day": date.day,
                          "min_distinct": tilelog.constants.MIN_DISTINCT_TILE_REQUESTS,
                          "min_requests": tilelog.constants.MIN_TILE_REQUESTS})
     click.echo("Writing tile usage to file")
     with lzma.open(dest, "w") as file:
         for tile in curs:
-            file.write("{}/{}/{} {}\n".format(tile[0], tile[1], tile[2], tile[3]).encode('ascii'))
+            tile_0, tile_1, tile_2, tile_3 = tile[0], tile[1], tile[2], tile[3]
+            file.write(f"{tile_0}/{tile_1}/{tile_2} {tile_3}\n").encode('ascii')
```

Comment:
I think this will work with the indexes in the f string

Reply:
Also, the ascii encoding was moved from inside the file.write call to outside, resulting in an error

Reply:
I have added that change.